pages/cart.page.js

```diff
@@ -0,0 +1,59 @@
+let WebPage = require("../base/webPage");
+
+let { WebButton } = require("../elemnets/button");
+let { WebView } = require("../elemnets/view");
+let { WebTextInput } = require("../elemnets/textInput");
+
+let baseElementLocator = by.css('#cart_title');
+let quantityFieldLocator = by.css('.cart_quantity_input');
+let quantityValueLocator = by.css('.cart_quantity :first-child[value="2"]');
+let checkoutButtonLocator = by.css('.cart_navigation [title="Proceed to checkout"]');
+let checkoutButtonLocator2 = by.css('[name="processAddress"]');
+
+
+class CartPage extends WebPage {
+    constructor() {
+        super();
+    }
+
+    async typeQuantity(number) {
+        await allure.createStep(`Clear "Qty" field, type ${number} in "Qty" field`, async () => {
+            await this.getQuantityFieldElement().clear();
+            await this.getQuantityFieldElement().sendKeys(number);
+        })();   
+    }
+
+    async checkQuantity() {
+        await this.getQuantityValueElement().waitForPresent();
+        return await this.getQuantityValueElement().isPresent();
```

Comment:
not logical part of code, it will never return `false` as it will fall on previous line

Reply:
Done